app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Article;
+use Illuminate\Http\Request;
+
+class ArticleController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // ④DBから記事情報を取得して変数に代入。
+        // $articles = Article::all();
+
+        // 更新した順に並び替え
+        $articles = Article::orderby('created_at', 'desc')->get();
+
+        //③記事一覧画面を表示。articleというリンク先（名前？）についてはlayouts/dashboardというものをviewさせるという意味になる！compactには上記の変数を入れており、DBからのデータを反映させている。
+        return view('layouts/dashboard', compact('articles'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+        // 記事投稿画面を表示
+        return view('layouts/create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //フォームに入力された内容を変数に取得
+        $form = $request->all();
+
+        // フォームに入力された内容をデータベースへ登録
+        $article = new Article();
+        $article->fill($form)->save();
+
+        // 記事一覧画面を表示
+        return redirect()->route('article.index');
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Article  $article
+     * @return \Illuminate\Http\Response
```

Comment:
```suggestion
     * @return \Illuminate\Http\Response
```
これもviewの型（上で指摘したやつ）に直しとこう

Reply:
<img width="784" alt="スクリーンショット 2022-12-05 23 26 56" src="https://user-images.githubusercontent.com/91363928/205662973-51a6d6aa-e94f-4722-aeca-2829eec71db3.png">
この画像の内容で合っているだろうか。

Reply:
@return \Illuminate\Http\Response\RedirectResponseに修正済。